ops/terraform/env/prod-sbx/stateful/main.tf

```diff
@@ -18,14 +18,14 @@ module "stateful" {
   aurora_config = {
     instance_class = "db.r5.2xlarge"
     cluster_nodes  = 3
-    engine_version = "11.9"
-    param_version  = "aurora-postgresql11"
+    engine_version = "12.6"
+    param_version  = "aurora-postgresql12"
   }
 
   aurora_node_params = [
     { name = "auto_explain.log_min_duration", value = "1000", apply_on_reboot = false },
-    { name = "auto_explain.log_verbose", value = true, apply_on_reboot = false },
-    { name = "auto_explain.log_nested_statements", value = true, apply_on_reboot = false },
+    { name = "auto_explain.log_verbose", value = "1", apply_on_reboot = false },
+    { name = "auto_explain.log_nested_statements", value = "1", apply_on_reboot = false },
```

Comment:
so it went from true to 1 and 0?

Reply:
Yes. When the terraform configs are applied, it will happily convert `true` to "1" ("1" represents `true` in the db parameter group file). But, the next time you run `terraform plan` or `terraform apply`, it sees that `true` != "1" so it wants to re-do the operation. Over and over and over. This just prevents that from happening.